importer/NoNo.py

```diff
@@ -1,53 +1,102 @@
 from Monument import Monument
 import importer_utils as utils
 
+MAPPING_DIR = "mappings"
+
 
 class NoNo(Monument):
-    """
-    A class to process monuments_no_(no).
 
-    TODO.
-    there's an API:
-    https://data.norge.no/data/riksantikvaren/kulturminnes%C3%B8k
-    look into how we can benefit from it!
-    """
+    def update_descriptions(self):
+        """
+        Add descriptions in several language.
 
-    def update_labels(self):
+        In the format "$type in $municipality, ($country if not nb)".
         """
-        Set the labels.
+        category = self.kategori.lower()
+        municip = self.get_municip_name()
+        base_norwegian = "{} i {}"
+        desc_norwegian = base_norwegian.format(category, municip)
+        self.add_description("nb", desc_norwegian)
+
+        base_english = "cultural property in {}, Norway"
+        desc_english = base_english.format(municip)
+        self.add_description("en", desc_english)
 
-        NOTE
-        Some of these are in all caps or have multiple spaces:
-        UTSIRA FYRSTASJON
-        SØGARD FJONE  -  FJONE SØNDRE
-        VÅLE PRESTEGÅRD, museum
+        base_swedish = "kulturarv i {}, Norge"
+        desc_swedish = base_swedish.format(municip)
+        self.add_description("sv", desc_swedish)
 
-        TODO
-        *Normalize - to title case?
-            It contains some old-style numbers, and these will be broken:
-            XXVIII -> Xxviii
+    def update_labels(self):
+        """
+        Add label in Bokmål.
 
-        * rm extra whitespace
+        Some of self.navn are in all caps.
+        These are normalized:
+        FÆRDER FYRSTASJON -> Færder Fyrstasjon
```

Comment:
Was the situation with multiple spaces resolved? How about the roman numerals? (based on your previous notes)